src/System.Drawing.Common/src/System/Drawing/Icon.cs

```diff
@@ -20,8 +18,7 @@ namespace System.Drawing
     ///     represent an object.  Icons can be thought of as transparent bitmaps, although
     ///     their size is determined by the system.
     /// </devdoc>
-    [Serializable]
-    public sealed class Icon : MarshalByRefObject, ISerializable, ICloneable, IDisposable
+    public sealed partial class Icon : MarshalByRefObject, ICloneable, IDisposable
     {
 #if FINALIZATION_WATCH
```

Comment:
For a future change, can we get rid of code under FINALIZATION_WATCH?  I wonder when the last time that constant was set and whether the code even still works.